dwave/cloud/auth/flows.py

```diff
@@ -202,3 +222,72 @@ def from_config_model(cls, config: ClientConfig, **kwargs) -> LeapAuthFlow:
             authorization_endpoint=authorization_endpoint,
             token_endpoint=token_endpoint,
             session_config=session_config)
+
+    def run_oob_flow(self):
+        """Run the OAuth 2.0 Authorization Code exchange flow, using the out of
+        band code exchange.
```

Comment:
```suggestion
        """Run OAuth 2.0 code exchange (out-of-band flow.) 
        
        Runs the OAuth 2.0 Authorization Code exchange flow using the out-of-band code 
        exchange.
```

For both methods the first sentence docstring won't fit the summary rendered in HTML